docs/source/app-dev/authorization.rst

```diff
@@ -61,8 +61,11 @@ Glossary:
 - A ``token`` (or ``access token``) is a tamper-proof piece of data that contains security information, such as the user identity or its privileges.
 - A ``token issuer`` is a service that generates tokens. Also known as "authentication server" or "Identity and Access Management (IAM) system".
 
+.. this anchor is kept around to not break existing links
```

Comment:
Is the anchor also rendered in HTML?

Reply:
It doesn't, thanks for asking. I'll revert all such changes.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6858/commits/a40534f8feb3aa8d303159b5f9823204f7687a35